backend/.env.example

```
@@ -93,6 +93,18 @@ MAX_UPLOAD_FILE_SIZE_MB=50
 MAX_EXTRACTED_TEXT_LENGTH=1000000
 
+# Attachment encryption configuration
+# Enable/disable AES-256-CBC encryption for attachment binary data (default: False)
+# When enabled, all newly uploaded attachments will be encrypted
+# Existing unencrypted attachments remain accessible (backward compatible)
+ATTACHMENT_ENCRYPTION_ENABLED=False
+# AES encryption key for attachments (32 bytes for AES-256)
+# SECURITY WARNING: Change these default values in production!
+# Generate secure random keys using: openssl rand -hex 32 (for key) and openssl rand -hex 16 (for IV)
+# These keys are separate from SHARE_TOKEN_AES_KEY for security isolation
+ATTACHMENT_AES_KEY=12345678901234567890123456789012
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ATTACHMENT_AES_KEY key should go before the ATTACHMENT_ENCRYPTION_ENABLED key

(UnorderedKey)